src/components/storybook-link/storybook-link.tsx

```diff
@@ -1,13 +1,14 @@
 import { FC, useEffect, useState } from 'react';
-import Link, { LinkProps } from '../link/link';
+import Link from '../link/link';
 import { hrefTo } from '@storybook/addon-links';
 import { LinkSize } from '../link/LinkConstants';
 import { withStaticProps } from '../../types';
 
-interface StorybookLinkProps extends LinkProps {
+interface StorybookLinkProps {
   page: string;
   children: string;
   story?: string;
+  size?: LinkSize;
 }
```

Comment:
```suggestion
import { FC, useEffect, useState } from 'react';
import Link, { LinkProps } from '../link/link';
import { hrefTo } from '@storybook/addon-links';
import { LinkSize } from '../link/LinkConstants';
import { withStaticProps } from '../../types';

interface StorybookLinkProps extends LinkProps {
  page: string;
  story?: string;
  href?: string;
}
```

This is also possible
That way you earn the LinkProps and still overriding href's required behavior

Reply:
But `href` isn't used, wouldn't it be confusing?

Reply:
Oh I thought you made the change only because it is required
So nm
so maybe you can do:

```suggestion
import { FC, useEffect, useState } from 'react';
import Link, { LinkProps } from '../link/link';
import { hrefTo } from '@storybook/addon-links';
import { LinkSize } from '../link/LinkConstants';
import { withStaticProps } from '../../types';

interface StorybookLinkProps {
  page: string;
  children: LinkProps["children"];
  story?: string;
  size?: LinkProps["size"];
}
```

To make sure it's understandable those props are related to Link

Reply:
But you can also leave it as is, it is not important